### PR TITLE
[DO NOT MERGE] Documentation Update for Azure/Mesos 

### PR DESCRIPTION
**Pull request by himanshu0503**
https://github.com/Shippable/docsv2/issues/307

### CODE REVIEW

File: sources/int_container_services.md [hunk 1]
```diff
@@ -272,3 +272,12 @@ To deploy containers to Docker Datacenter, you need to configure an account inte
...
 * Click on `Add Integration`. Choose `Docker Datacenter`, name your integration and provide the username, password and URL to access the Universal Control Plane.
 * Click `Save`.
+
+<a name="acs-integration"></a>
+## Microsoft Azure Container Service
+To deploy containers to Docker Microsoft Azure Container Service, you need to configure an account integration with credentials to access your Microsoft Azure Container Service account.
+
+* Login to your azure account and search your Mesos Master VM's `LINUXADMINUSERNAME` and `MASTERFQDN`.
+* Sign in to Shippable and go to your [Account Settings](https://app.shippable.com/accountSettings). Click on the `Integrations` tab.
+* Click on `Add Integration`. Choose `Azure Container Service`, name your integration and provide the username (LINUXADMINUSERNAME) and URL(MASTERFQDN) to access the Mesos Master VM.
```
Comment (himanshu0503) [inline sources/int_container_services.md]: The masterIntegration's display name is `Azure Container Service`, should we change it to `Microsoft Azure Container Service`?

**Comment by manishas** [comment]
there are some things still being ironed out here so we shouldn't merge this.

**Comment by avinci** [comment]
@manishas should this be merged?

**Comment by avinci** [comment]
we need to redo this

**Pull request closed without merging**